src/ui/shared/openUrl.ts

```diff
@@ -0,0 +1,27 @@
+import browser from 'webextension-polyfill';
+import { urlContext, type UrlContextParams } from './UrlContext';
+
+async function getNextToActiveTabIndex() {
+  const [activeTab] = await browser.tabs.query({
+    active: true,
+    lastFocusedWindow: true,
+  });
+  return activeTab?.index + 1;
```

Comment:
```suggestion
  return (activeTab?.index ?? 0) + 1;
```

Reply:
Although maybe it's better to return `null` and handle it in the call site

Reply:
I prefer your suggested change. The caller would always want to fall back to tab index = 0 anyway